src/Events/Program.cs

```diff
@@ -201,10 +201,20 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
 
     if (config.GetValue<bool>("PostgreSQLSettings:EnableDBConnection"))
     {
-        services.AddHealthChecks()
-            .AddNpgSql(string.Format(
+        var connectionString = string.Format(
                 config.GetValue<string>("PostgreSQLSettings:ConnectionString"),
-                config.GetValue<string>("PostgreSQLSettings:EventsDbPwd")));
+                config.GetValue<string>("PostgreSQLSettings:EventsDbPwd"));
+
+        services.AddHealthChecks()
+            .AddNpgSql(connectionString);
+
+        services.AddNpgsqlDataSource(connectionString, builder => builder
+                .EnableParameterLogging(false)
```

Comment:
I would suggest to enable parameterlogging in general and then filter which statements to log in the ".ConfigureCommandFilter" 4 lines below. Replace true with a test on cmd.CommandText (unless it's too complex to specify an appropriate filter). NB the the test determines what to exclude - not what to include. This also requires that the log level for npgsql is set to Information.